web/src/components/Navbar/index.js

```diff
@@ -1,88 +1,99 @@
+            </svg>
+          </a>
+          <div className="nav__list">
+            <div className="nav__list">
+            <a href="mailto:ellie@colark.com">Contact</a>
+            </div>
+            {navbarItemsList}
+          </div>
+          <button className="nav__menu" onClick={ this.props.operation}>
```

Comment:
I think the file you had in gatsby was probably perfect, but until we have the hamburger button working in unstack, could you be sure to hide it?

Reply:
@alodahl it's hidden with display:none in the media query but would you prefer me taking it out entirely?

Reply:
that's fine, I didn't realize that.